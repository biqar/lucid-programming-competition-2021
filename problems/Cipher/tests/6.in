5
2 3 1 5 4
Lucid Software's core values are: teamwork over ego; innovation in everything we do; individual empowerment, initiative, and ownership; and passion and excellence in every area.