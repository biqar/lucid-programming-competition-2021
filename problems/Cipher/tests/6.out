uSa  eeakr;oo ygdndeetiiawhaan lee .corcvs:m   vnet odumr,tvnnins el vaL wseurereoninrn ii wnnt os podec yadt'rlatovgntieie vloeia, r; incnnreifeoa  woeia vhw;iapm iedepdsaxeier